Bill type description correction (#579)
MewsSystems/gitbook-connector-api: operations/bills.md

#### Bill type

> In default, a bill is `Receipt` indicating that it has been partially or fully paid or has not yet been paid. Alternatively, it may be `Invoice` that is supposed to be paid in the future.

Reviewer: Did you ask the domain owners to explain the difference between these two types?

When looking at the source code, it seems that the concept of Invoice is more concrete while Receipt is a default type used for everything else. Also, I found [this validation rule](https://github.com/MewsSystems/mews/blob/48f21667d6ad97f2492f0829b44f14c4633cecca/src/Modules/Core/Mews.Business/Finance/Documents/Bills/BillComponent.cs#L541) which says that only invoices can be marked as paid. I think we don't need to base the distinction between bill types on when it is paid, there must be something else.

> In default, a bill is `Receipt` indicating that it has been partially or fully paid or has not yet been paid. Alternatively, it may be `Invoice` that is supposed to be paid in the future.

Reviewer 2: ```suggestion
After a bill is closed, the Bill Type is set to `Receipt` or `Invoice`. `Receipt` indicates that the bill has been fully paid and the balance is zero. `Invoice` indicates that the bill has not yet been fully paid but an invoice has been issued. Prior to closing, Bill Type should not be used.
```

> * `Receipt` - the bill has been partially or fully paid or has not yet been paid.

Reviewer 2: ```suggestion
* `Receipt` (default) - the bill has been paid in full; only applicable after the bill is closed
```

> * `Invoice` - the bill is supposed to be paid in the future. Before closing it is balanced with an invoice payment.

Reviewer 2: ```suggestion
* `Invoice` - the bill has not been paid in full but an invoice has been issued to request payment
```
